Failed to generate HTML from arc42 template: SEVERE: arc42.adoc: line 9: include file not found

After upgrading to Gradle 7.5.1 and Asciidoc-Gradle-Plugin v4 we get an error when generating HTML from the arc42 templates.

**To Reproduce**
Steps to reproduce the behavior:
1. Create a test directory and Install `dtcw` from the github repo ng
```
mkdir test && cd test
curl -Lo dtcw https://raw.githubusercontent.com/docToolchain/docToolchain/ng/dtcw
chmod +x dtcw
```
2. Install the current `ng` branch of docToolchain
```
export DTC_VERSION=latest
./dtcw install doctoolchain
...
# Note - this will lead to an error which you can ignore (Task 'doctoolchain' not found)
```
3. Download arc42 template
```
./dtcw downloadTemplate
```
5. Generate HTML of arc42 template
```
./dtcw generateHTML
dtcw - docToolchain wrapper V0.38
docToolchain Vlatest
docker available
home folder exists
force latest version with local install
Already up to date.
use local homefolder install /home/max/.doctoolchain/
Java Version 11
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.5.1/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 

> Task :generateHTML
Converting /home/max/work/dtc/test-dtcw-2.3.0/src/docs/arc42/arc42.adoc
Apr 26, 2023 9:04:28 PM uri:classloader:/gems/asciidoctor-2.0.18/lib/asciidoctor/reader.rb preprocess_include_directive
SEVERE: arc42.adoc: line 9: include file not found: /home/max/work/dtc/test-dtcw-2.3.0/src/docs/chapters/config.adoc
include file not found: /home/max/work/dtc/test-dtcw-2.3.0/src/docs/chapters/config.adoc :: arc42.adoc :: /home/max/work/dtc/test-dtcw-2.3.0/src/docs//home/max/work/dtc/test-dtcw-2.3.0/src/docs/arc42/arc42.adoc:9 (uri:classloader:/gems/asciidoctor-2.0.18/lib/asciidoctor/reader.rb:preprocess_include_directive)
Apr 26, 2023 9:04:28 PM uri:classloader:/gems/asciidoctor-2.0.18/lib/asciidoctor/parser.rb parse
SEVERE: arc42.adoc: line 11: level 0 sections can only be used when doctype is book
....
Exception in thread "main" org.asciidoctor.gradle.remote.AsciidoctorRemoteExecutionException: ERROR: The following messages from AsciidoctorJ are treated as errors:
image to embed not found or not readable: /home/max/work/dtc/test-dtcw-2.3.0/src/images/arc42-logo.png
        at org.asciidoctor.gradle.remote.ExecutorBase.failOnWarnings(ExecutorBase.groovy:231)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
        at org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)
        at org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)
        at org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)
        at org.asciidoctor.gradle.remote.AsciidoctorJavaExec$_run_closure3.doCall(AsciidoctorJavaExec.groovy:76)
        at org.asciidoctor.gradle.remote.AsciidoctorJavaExec$_run_closure3.call(AsciidoctorJavaExec.groovy)
        at org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2358)
        at org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2343)
        at org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2384)
        at org.asciidoctor.gradle.remote.AsciidoctorJavaExec.run(AsciidoctorJavaExec.groovy:68)
        at org.asciidoctor.gradle.remote.AsciidoctorJavaExec.main(AsciidoctorJavaExec.groovy:49)

> Task :generateHTML FAILED

FAILURE: Build failed with an exception.
```

**Expected behavior**
The HTML page is generated.

**Additional Context**
Using docToolchain 2.2.1 works
```
unset DTC_VERSION
./dtcw generateHTML
dtcw - docToolchain wrapper V0.38
docToolchain V2.2.1
docker available
home folder exists
use local homefolder install /home/max/.doctoolchain/
Java Version 11
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/6.9.2/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 

BUILD SUCCESSFUL in 6s
1 actionable task: 1 up-to-date
```

**Configuration**
 - docToolchain version: commit dd8750fe
 - OS: Linux, Debian unstable
 - Browser: irrelevant
 - Relevant SDK Versions: bug replicable with JDK 8 and 11.

--- a/scripts/AsciiDocBasics.gradle
+++ b/scripts/AsciiDocBasics.gradle
@@ -174,7 +174,7 @@ asciidoctorj {
 // this is needed for PDF generation with plantUML
 tasks.withType(AbstractAsciidoctorTask) { docTask ->
 
-    baseDirFollowsSourceDir()
+    baseDirFollowsSourceFile()
 
     config.taskInputsDirs.each {
         inputs.dir new File(docDir, it)

